.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -28,37 +28,52 @@ jobs:
   linux_x64_clang:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'linux_x64_clang')
-    runs-on:
-      - self-hosted # must come first
-      - runner-group=${{ needs.setup.outputs.runner-group }}
-      - environment=${{ needs.setup.outputs.runner-env }}
-      - cpu
-      - os-family=Linux
+    runs-on: azure-linux
```

Comment:
yeah sorry, I clicked create PR without changing the destination.  I'll fix all that once I make sure all the tests are passing

Reply:
Looks like @saienduri changed the name to `azure-linux-scale` in https://github.com/iree-org/iree/commit/9dc6b904f9125c8e407ef0a23b81e93c0a4757d1. More reason to base off of the shared branch instead of `main`.

Right now this PR is waiting for a runner that won't spin up: https://github.com/iree-org/iree/actions/runs/10729356454/job/29755788260?pr=18448